### DISCUSSION TITLE
cant start suricata and strelka

### DISCUSSION THREAD

**Discussion by J0hnnyb0y86** [Security-Onion-Solutions/securityonion#5545] [Unsupported Versions]
hi all when i test suricata start or stop i receive this
=========================================================================
Stopping suricata...
=========================================================================
Error response from daemon: No such container: so-suricata
Error: No such container: so-suricata
-------------------------------------------------------------------------------------------------------------------------------------
[ERROR   ] Failure!
local:
----------
          ID: suricata_state_not_allowed
    Function: test.fail_without_changes
      Result: False
     Comment: Failure!
     Started: 16:24:54.029744
    Duration: 1.569 ms
     Changes:   

Summary for local
------------
Succeeded: 0
Failed:    1
------------
Total states run:     1
Total run time:   1.569 m

can i download manually docker image?

**Comment by dougburks** [comment]
If it says `suricata_state_not_allowed`, then that's based on how you configured the installation. What options did you choose when running Setup?
